source/index.html.md

```diff
@@ -82,3 +83,60 @@ There are also contextual links to other related content resources contained wit
+```
+
+#### Format specific response fields.
+
+Different styles of content on GOV.UK are displayed in distinct _formats_.
+
+For example, simple guidance on current VAT rates can be succinctly displayed in a single page format.
+Whereas [The Highway Code ](https://www.gov.uk/guidance/the-highway-code) is ordered into multiple sections across several pages.
+
+The GOV.UK Content API exposes these format specific differences in a discrete field called `details`.
+
+In the example above the `body` field containing the content body is embedded in the `details` field.
+
+Compare this to the `details` field from the API response for The Highway Code:
+
+```
+curl -X GET https://www.gov.uk/api/content/guidance/the-highway-code -H 'Accept: application/json' | json_pp
+```
+
+```
+"details": {
+  "body": "\n<div role=\"note\" aria-label=\"Information\" class=\"application-notice info-notice\">\n<p>There&rsquo;s a different version of The Highway Code for <a rel=\"external\" href=\"https://www.nidirect.gov.uk/articles/highway-code\">Northern Ireland</a>.</p>\n</div>\n\n<p>You can <a rel=\"external\" href=\"http://www.safedrivingforlife.info/shop/product/official-highway-code-new-edition-2015\">buy The Highway Code</a> from the Safe Driving for Life website, or from most high street and online bookshops, app and eBook stores...",
+  "child_section_groups": [
+  {
+    "title": "Contents",
+    "child_sections": [
+    {
+      "title": "Introduction",
+      "description": "Who The Highway Code is for, how it's worded, and the consequences of not following the rules. ",
+      "base_path": "/guidance/the-highway-code/introduction"
+    },
+    {
+      "title": "Rules for pedestrians (1 to 35)",
+      "description": "The Highway Code rules for pedestrians, including general guidance, crossing the road, crossings, and situations needing extra care.",
+      "base_path": "/guidance/the-highway-code/rules-for-pedestrians-1-to-35"
+    },
+    {
+      "title": "Rules for users of powered wheelchairs and mobility scooters (36 to 46) ",
+      "description": "The Highway Code rules for powered wheelchairs and mobility scooters, including on pavements and on the road.",
+      "base_path": "/guidance/the-highway-code/rules-for-users-of-powered-wheelchairs-and-mobility-scooters-36-to-46"
+    },
+  ...
+```
+
+The chaptered content of this piece of guidance is split into multiple pages and the API response is structured so that these pages, or content resources,
+can be visited at a distinct path.
+
+#### Metadata fields
+
+While the `details` fields of the API response may differ per format, the _metadata_ fields, those at the top level of the JSON response, are generally very similar as they describe the content resource, eg. when the item was last updated.
+
+The [reference documentation](/govuk-content-api-docs/reference.html#contentitem) contains much more detail on each of the fields in the API response.
```

Comment:
Similar to before, I wonder if we can just link to `reference.html` to avoid the full path.